nixos/release.nix

```diff
@@ -265,27 +264,17 @@ in rec {
 
     with import ./.. { inherit system; };
 
-    hydraJob ((import lib/eval-config.nix {
-      inherit system;
-      modules =
-        [ configuration
-          versionModule
-          ./maintainers/scripts/ec2/amazon-image.nix
-        ];
+    hydraJob ((evalSystemConfiguration {
+      modules = makeModules system ./maintainers/scripts/ec2/amazon-image.nix {};
```

Comment:
These and the following where actually hidden `makeModules` incantations, so made that explicit. Sry for the extra diff... Semantics are semantics.